ecommerce/app/views/customers/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<p id="notice"><%= notice %></p>
+<h1>Customers</h1>
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Email</th>
+      <th>City</th>
+      <th>Pincode</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @customers.each do |customer| %>
+      <tr>
+        <td><%= customer.name %></td>
```

Comment:
Reduce your controller query using select based on the fields you are using here